components/shared/OnRampModal.vue

```diff
@@ -115,17 +116,22 @@ const getImage = (service: string) => {
 const getSupportedTokensToText = (tokens: string[]) =>
   tokens.map(token => `$${token}`).join(', ')
 
-const providers = computed(() => [
+const PROVIDER_TOKEN_SUPPORTS = {
+  SUB: ['DOT', 'KSM'],
+  EVM: ['ETH'],
+}
+
+const providers = computed<{ image: string, disabled: boolean, supports: Record<ChainVm, string[]>, value: Provider }>(() => [
```

Comment:
no need manual assign the types, it will return types automatically, no?
`computed<{...}>()` to `computed()`

Reply:
yes, but this way the computed is forced to match that type if an item fails to match I can notice it